README.md

```diff
@@ -422,6 +425,153 @@ Once you have the in-process link, you can create an in-process link tracer to c
+receiving one or multiple messages if you are interested in the time the actual receiving takes. For the processing of the message, i.e.,
+any business logic that is executed in response to the message content, there is a different tracer. Note that only the tracer for
+processing messages supports an incoming tag. This is because the incoming tag must be set before starting the tracer and this cannot be
+done for the receive-tracer, as the tag is normally part of the message content or headers, thus available only after message receipt.
+
+The following example shows using the message receive and process tracers in the recommended combination, i.e., starting the process tracer
+inside the receive tracer:
+
+```C
+    onesdk_tracer_handle_t receive_tracer = onesdk_incomingmessagereceivetracer_create(messagingsysteminfo_handle);
+
+    /* start receive_tracer */
+    onesdk_tracer_start(receive_tracer);
+
+    while (/* ... e.g., a message is available in the queue, or only once ... */) {
+        /* ... actually receive a message, ... */
+
+        unsigned char const* byte_tag = ...;    /* pointer to the byte tag that we received as part of the message, from the sender  */
+        onesdk_size_t byte_tag_size = ...;      /* size of the byte tag that we received */
+
+        /* create process_tracer */
+        onesdk_tracer_handle_t const process_tracer = onesdk_incomingmessageprocesstracer_create(messagingsysteminfo_handle);
+
+        /* set the tag that we got from the message */
+        if (byte_tag_size != 0)
+            onesdk_tracer_set_incoming_dynatrace_byte_tag(process_tracer, byte_tag, byte_tag_size);
+
+        /* optional: set message ID, if provided by the messaging system */
+        onesdk_outgoingmessagetracer_set_vendor_message_id(process_tracer, onesdk_asciistr(mymessage_get_id_str(mymessage)));
+
+        /* optional: set correlation ID, if provided by the messaging system */
+        onesdk_outgoingmessagetracer_set_correlation_id(process_tracer, onesdk_asciistr(mymessage_get_correlation_id_str(mymessage)));
+
+        /* start process_tracer */
+        onesdk_tracer_start(process_tracer);
+
+        /* ... do the actual work: process the message content, do something in response, ... */
+
+        /* set error information */
+        if (something_went_wrong_with_processing)
+            onesdk_tracer_error(process_tracer, onesdk_asciistr("error type"), onesdk_asciistr("error message"));
+
+        /* end & release process_tracer */
+        onesdk_tracer_end(process_tracer);
+    }
+    /* set error information */
+    if (something_went_wrong_with_receiving)
+        onesdk_tracer_error(receive_tracer, onesdk_asciistr("error type"), onesdk_asciistr("error message"));
+    onesdk_tracer_end(receive_tracer);
+```
+
+Note that currently, the receive tracer will never create a new PurePath, so if it is not started inside another started tracer, nothing
```

Comment:
until now, we never mentioned such implementation details ... @SonjaChevre ?

Reply:
I would also remove it from here

Reply:
This is not an implementation detail, it is very user-visible. Also, contrary to the DB tracer, this is by design here. See also #4.